test/data/simulated.minimap2.hg19.coordsorted_cutoff.sam

```diff
@@ -1,3 +1,5 @@
+@HD	VN:1.6	SO:coordinate
+@SQ	SN:chr21	LN:46709983
```

Comment:
If you make changes in our test data you have to update the shasum in the datasources.cmake

Reply:
Done!